sharepoint/sharepoint-ps/sharepoint-online/Remove-SPODeletedContainer.md

```diff
@@ -0,0 +1,97 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: 
+applicable: SharePoint Online
+title: Remove-SPODeletedContainer
+schema: 2.0.0
+author: cindylay
+ms.author: cindylay
+ms.reviewer:
+---
+
+
+# Remove-SPODeletedContainer
+
+
+
+## SYNOPSIS
+
+Permanently deletes the specified Container from the Recycle Bin. This action cannot be undone.
+
+## SYNTAX
+
+
+
+### ParamSet1
+
+```powershell
+Remove-SPODeletedContainer [–Identity <ContainerId>] [<CommonParameters>]
+```
+
+
+
+## DESCRIPTION
+
+The `Remove-SPODeletedContainer` cmdlet permanently removes a deleted Container from the Recycle Bin. A permanently deleted Container cannot be recovered. You must be a SharePoint Administrator or Global Administrator to run the cmdlet.
+
+:warning:  **Please Note:**  Permanently deleting Containers is generally NOT recommended
+
+ 
+
```

Comment:
```suggestion
The `Remove-SPODeletedContainer` cmdlet permanently removes a deleted container from the Recycle Bin. A permanently deleted container cannot be recovered. You must be a SharePoint Administrator or Global Administrator to run the cmdlet.

**Important**: We do not generally recommend that you permanently delete containers.
```

Reply:
We are considering capitalizing the term Container as a SharePoint repository services Container across all our docs. Would it be ok to capitalize in that scenario?